flox-bash/lib/commands.sh

```diff
@@ -108,6 +108,43 @@ function parseNixArgs() {
 	done
 }
 
+# This function parses out common flake options used by the flox
```

Comment:
Should 84b83d2dfa8486ea2caf62cd5a418d2199d6eec8 be in a separate PR?

Reply:
It could be merged separately but it has to come first because it's not possible to publish from anything but the default branch for a repo without it.

Reply:
Given the description and name of the PR I wouldn't expect this change in this PR..

Aside, I still don't think we should incorporate all the nix flags into the flox cli and we should